assets/dev/scss/global/_aspect_ratio.scss

```diff
@@ -1,3 +1,5 @@
+
+// TODO: Remove this method because it has been replaced by native aspect ratio property in v3.11
```

Comment:
```suggestion
// TODO: Remove this method because it has been replaced by native aspect ratio property in v3.11
```

Reply:
Maybe put it in the backlog, so we'll actually do it.
Plus, maybe we need to tell 3rd party that we're gonna remove it?

Reply:
Sure.